client/mm/mm_basic.go

```diff
@@ -638,36 +839,74 @@ func (m *basicMarketMaker) run() {
 		}
 	}()
 
-	wg.Wait()
+	// Periodically update asset fee rates
+	wg.Add(1)
+	go func() {
+		defer wg.Done()
+		refreshTime := time.Minute * 10
+		for {
+			select {
+			case <-time.NewTimer(refreshTime).C:
+				err := m.updateFeeRates()
```

Comment:
The only arguments to `MaxFundingFees` are `numTrades` and `options`. I don't see a reason to run this on a timer. We need to update these only if one of those values changes.

Reply:
I've updated `MaxFundingFees` to use the server's `MaxFeeRate` instead of the wallet's fee rate limit.